source/platforms/nvidia-ai-workbench.md

```diff
@@ -0,0 +1,63 @@
+# NVIDIA AI Workbench
+
+[NVIDIA AI Workbench](https://www.nvidia.com/en-us/deep-learning-ai/solutions/data-science/workbench/) is a developer toolkit for data science, machine learning, and AI project development that lets you develop on your laptop/workstation and then move workloads to scalable GPU resources in a data center or the cloud. AI Workbench is free, you can install it in minutes on your local or remote computers, and it provides both a desktop application and a command-line interface (CLI).
+
+## Installation
+
+You can install AI Workbench locally, or on a remote computer that you have SSH access to.
+
+Follow the [AI Workbench installation](https://docs.nvidia.com/ai-workbench/user-guide/latest/installation/overview.html) documentation for instructions on installing on different operating systems.
+
+```{admonition} Thick client
```

Comment:
The thick/thin client is a concept I wasn't aware of, I had to read both to sort of get it. I wonder if this reduce to GPU on local / GPU on remote. Maybe making this more clear in the titles?

Reply:
Thanks for calling this out. Reducing jargon is good, I've gone with "Local GPU System" and "Remote GPU System" instead which hopefully adds clarity.